operatorcert/entrypoints/create_container_image.py

```diff
@@ -0,0 +1,150 @@
+import argparse
+import logging
+from typing import Any
+from urllib.parse import urljoin
+
+from operatorcert import pyxis
+from operatorcert.logger import setup_logger
+
+LOGGER = logging.getLogger("operator-cert")
+
+
+def setup_argparser() -> Any:
+    """
+    Setup argument parser
+
+    Returns:
+        Any: Initialized argument parser
+    """
+    parser = argparse.ArgumentParser(description="Bundle dockerfile generator.")
+
+    parser.add_argument(
+        "--pyxis-url",
+        default="https://catalog.redhat.com/api/containers/",
```

Comment:
This script will be used with internal Pyxis. Can you change the default to internal route?